### ISSUE TITLE
Login Fails API error: Record not found

### ISSUE THREAD

**Issue by slrgt** [Nutomic/ibis#20]
I created my account and successfully edited a page (the announcing page had a mistake in the link to open.ibis.wiki) so I know my account works. but if i open a new tab and go to ibis.wiki it asks me to login and if i do it says API error: Record not found. This happens if i try to login in an incognito window as well so I know its not because I'm logged in another tab.

**Comment by Nutomic** [comment]
Yes the account is only for the particular instance where you signed up (just like on Lemmy).
